app/assets/javascripts/client/controllers/learning_questions/LearningQuestionListDisplayCtrl.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('LearningQuestionListDisplayCtrl', LearningQuestionListDisplayCtrl);
+
+  LearningQuestionListDisplayCtrl.$inject = [
+      '$stateParams',
+      '$scope',
+      '$window',
+      'LearningQuestion'
+  ];
+
+  function LearningQuestionListDisplayCtrl($stateParams, $scope, $window, LearningQuestion) {
+    var vm = this;
+    vm.learningQuestions = [];
+
+    vm.extractId = function() {
+      return $stateParams.assessment_id || $stateParams.id;
+    };
+
+    vm.loadQuestions = function () {
+      LearningQuestion
+          .get({assessment_id: vm.extractId()})
+          .$promise
+          .then(function (result) {
+            vm.learningQuestions = result.learning_questions;
+          });
+    };
+
+    vm.deleteLearningQuestion = function (model) {
+      var willDelete = $window.confirm('Are you sure you wish to delete this learning question?');
+      if (willDelete) {
```

Comment:
You're performing a decision here that affects the rest of the scope, for such cases I prefer stopping the function execution ASAP.
```js
if (!willDelete) { return; }
```
This way if there is anything else we'd like to add in the future we'll add it to the end of the function instead the `if` branch, looks cleaner to me.